scania-digital-design-system/tegel
# feat(header): prefix change

**Describe pull-request**  
Prefix change for the header component.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1638

**How to test**  
1. Go to storybook
2. Check in Components -> Header
3. Check in Patterns -> Navigation
4. Make sure the style is correct and check the API.
